src/profile/webid.test.ts

```diff
@@ -170,44 +176,49 @@ describe("getProfileAll", () => {
       mockSolidDatasetFrom(MOCK_WEBID),
       profileContent
     );
-    // Mock the webid unauthenticated fetch
-    const mockedUnauthFetch = jest.requireMock("cross-fetch") as {
-      fetch: jest.Mocked<typeof fetch>;
-    };
-    mockedUnauthFetch.fetch.mockResolvedValueOnce(
-      new Response(await triplesToTurtle(toRdfJsQuads(webIdProfile)), {
+    const mockedProfileResponse = mockResponse(
+      await triplesToTurtle(toRdfJsQuads(webIdProfile)),
+      {
         headers: {
           "Content-Type": "text/turtle",
         },
-        url: MOCK_WEBID,
-      } as ResponseInit)
+      },
+      MOCK_WEBID
     );
 
+    mockedFetch
+      .mockResolvedValueOnce(mockedProfileResponse)
+      .mockResolvedValueOnce(mockedAltProfileResponse);
+
     await getProfileAll(MOCK_WEBID);
-    // The embedded fetch should have been used to fetch the alt profile.
-    expect(mockedAuthFetcher.fetch).toHaveBeenCalledTimes(1);
-    expect(mockedAuthFetcher.fetch).toHaveBeenCalledWith(
-      "https://some.profile",
+    // The embedded fetch should have been used to fetch the WebID Profile first,
+    // and then the alt profile.
```

Comment:
Why was this not happening before?

Reply:
The implementation changed a bit to move from using the global fetch to using the bound default fetch, so the mocks needed to be slightly changed.